src/unix/udp.c

```diff
@@ -349,7 +349,11 @@ int uv__udp_bind(uv_udp_t* handle,
 static int uv__udp_maybe_deferred_bind(uv_udp_t* handle,
                                        int domain,
                                        unsigned int flags) {
-  unsigned char taddr[sizeof(struct sockaddr_in6)];
+  union {
+    struct sockaddr_in6 in6;
+    struct sockaddr_in in;
+    struct sockaddr addr;
+  } taddr;
```

Comment:
Just asking :-) The union works, of course.  We do use `struct sockaddr_storage` in many other places IIRC. No biggie though!